labcontrol/gui/templates/plate.html

```diff
@@ -159,31 +159,46 @@ <h4>Plate map</h4>
 
     <div id='plate-map-div' class='spreadsheet-container'></div>
 
-    <!-- Plate map legend div -->
+    <!-- Plate map legend div
+        Note that well-unknown and well-prev-plated are mutually exclusive
+        while well-duplicated is not. A duplicated value can appear in the
+        plate and be valid, unknown, or previously plated on another plate. -->
     <div id='plate-map-legend'><h5>Plate map legend</h5>
         <div id='legend-rows' class='legend-container'>
           <section id='plate-map-legend-commented' class='legend-entry'>
-            <div id='plate-map-legend-commented-box' class='well-commented legend-patch'></div>
+            <div id='plate-map-legend-commented-box' class='well-commented legend-patch'><center>Specimen ID</center></div>
             <div id='plate-map-legend-commented-text' class='legend-text'>
-              <p> The plate well has a comment.</p>
+              <p> Plate well has an attached comment</p>
             </div>
           </section>
           <section id='plate-map-legend-duplicated' class='legend-entry'>
             <div id='plate-map-legend-duplicated-box' class='well-duplicated legend-patch'></div>
             <div id='plate-map-legend-duplicated-text' class='legend-text'>
-              <p> The Sample ID already exists on this plate.</p>
+              <p> Specimen ID duplicated on current plate</p>
             </div>
           </section>
           <section id='plate-map-legend-prev-plated' class='legend-entry'>
             <div id='plate-map-legend-prev-plated-box' class='well-prev-plated legend-patch'></div>
             <div id='plate-map-legend-prev-plated-text' class='legend-text'>
-              <p> The Sample ID has been plated previously on another plate.</p>
+              <p> Specimen ID plated previously on another plate</p>
             </div>
           </section>
           <section id='plate-map-legend-unknown' class='legend-entry'>
             <div id='plate-map-legend-unknown-box' class='well-unknown legend-patch'></div>
             <div id='plate-map-legend-unknown-text' class='legend-text'>
-              <p> The Sample ID does not currently exist in any studies.</p>
+              <p> Specimen ID is not a member of the attached studies</p>
+            </div>
+          </section>
+          <section id='plate-map-legend-indeterminate' class='legend-entry'>
+            <div id='plate-map-legend-indeterminate-box' class='well-indeterminate legend-patch'></div>
+            <div id='plate-map-legend-indeterminate-text' class='legend-text'>
+             <p> Specimen ID matches more than one member of the attached studies</p>
+            </div>
+          </section>
+          <section id='plate-map-legend-indeterminate' class='legend-entry' style="visibility: hidden">
```

Comment:
I'm unclear on why there's an invisible `<section>` here. I'm guessing this is a hack to make the layout work properly? (I have no room to judge on those sorts of things, don't worry :)

At the very least, it would be a good idea to document *why* this invisible `<section>` exists so that future developers aren't confused.

Reply:
I'm not at my computer right now so I can't check, but I would imagine this is to make the legend centered on the last row. @charles-cowart if confirmed, I would reccomend something like:
```
<!-- Include the section below if there are an odd number of legend entries -->
<section id='plate-map-legend-for-spacing class='legend-entry' style="visibility: hidden">
  <div id='plate-map-legend-for-spacing' class='legend-patch'></div>
  <div id='plate-map-legend-for-spacing' class='legend-text'></div>
</section>
```
There might be a more minimal solution.

Reply:
I'm going to document it as is - basically the hidden section is there because otherwise the 'Well comments' header is offset to the middle. I'll leave it to you guys to decide if it's worth replacing in a subsequent PR.

Reply:
Sounds good to me. As long as the code is documented, I don't think replacing this in the future is important.